website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+
+To make it simpler to access PactFlow, we've rolled out Single Sign On (SSO) authentication with Github, Google and SAML2.0.
+
+## How it works
+
+SSO users are automatically provisioned when they first authenticate.
+
+**NOTE:** &nbsp; Changing these settings may require users to do a hard refresh in their browser (CTRL-F5 on Windows and Linux, CMD-R on Mac OSX) before the next time they try to login, or clear their browser cookies and cache.
+
+## Prerequisites
+
+If you are migrating to Single Sign On from username/password based authentication, you should consider enabling merging of identities based on email address, via the ["Consolidate User Logins by Email" system preference](/docs/user-interface/settings/preferences#consolidate-user-logins-by-email), otherwise you will end up with duplicate users.
+
+## GitHub Authentication
+
+_Eligible plans: all_
+
+For GitHub authentication, you need to configure the GitHub organisations that you want users who belong to be able to log into your broker instance.
+
+Steps to configure
+
+1. Navigate to the Authentication settings page and add the name of the Github organization(s) that should be granted access (_hint_: given this project URL the name should be `pactflow`: github.com/pactflow/example-project). Make sure you get the case right, as the name will need to match exactly.
+1. Press the Save button.
+1. After configuring Github, you will need to log out.
+1. Once logged out navigate back to your login screen. You should now have a Github option.
+1. Login with the new Github option.
+1. On the first authentication attempt, it will require Github authorization. Follow the in-app prompts to guide you.
+
+![GitHub Authentication Settings](/ui/github-auth.png)
+
+### Troubleshooting
+
+#### Seeing error "None of your GitHub organizations have been registered with PactFlow." when trying to log in
+
+There are multiple reasons why this error may be shown.
+
+##### You may not have clicked "Grant" or "Request" for that organization on the GitHub PactFlow Authorize screen
+
+To fix this, log out of PactFlow, go to https://github.com/settings/applications. Revoke PactFlow access, then log in again. You will be given the opportunity to grant or request access to your organization again during the login flow.
+
+
+##### If you clicked "Request", you may need to ask an administrator to approve the authorization
+
+To fix this, please contact your organization's GitHub administrator, and ask that they follow these instructions to approve PactFlow https://help.github.com/en/articles/approving-oauth-apps-for-your-organization
+
+##### You may have previously authenticated to PactFlow with GitHub, but joined the organization since doing so
+
+To fix this, log out of PactFlow, go to https://github.com/settings/applications. Revoke PactFlow access, then log in again. You will be given the opportunity to grant or request access to your organization again during the login flow.
```

Comment:
To fix this, log out of PactFlow, and go to

 You will get an opportunity to grant or request access to your organization again during the login flow.